ray-operator/config/samples/ray-cluster.mini.yaml

```diff
@@ -53,6 +53,15 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
+          resources:
+            limits:
+              cpu: 2
+              memory: 4Gi
+              gpu: 1
```

Comment:
I think mini is being used by many users to verify the control plane functionality. I would suggest to use less resources like 1c 2gi and remove gpu. WDYT?

Reply:
fix it

Reply:
gpu is still there.